website/docs/r/cdn_frontdoor_custom_domain_association.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "CDN"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_cdn_frontdoor_custom_domain_association"
+description: |-
+  Manages the single apply/single destroy incompatibility between the CDN FrontDoor API and the Terraform dependency graph.
+---
+
+# azurerm_cdn_frontdoor_custom_domain_association
+
+Manages the single destroy incompatibility between the CDN FrontDoor API and the Terraform dependency graph generation. You should use this association resource to avoid receiving the `This resource is still associated with a route. Please delete the association with the route first before deleting this resource` error when issuing the Terraform `destroy` command.
```

Comment:
this links a FrontDoor to a Custom Domain Association? that the API returns an error signals this was a design issue that we've fixed, not something users should need to consider, so I think we can simplify this to:

```suggestion
  Manages the association between a CDN FrontDoor Custom Domain and one or more CDN FrontDoor Routes.
 ---

 # azurerm_cdn_frontdoor_custom_domain_association

Manages the association between a CDN FrontDoor Custom Domain and one or more CDN FrontDoor Routes.
```